### PR TITLE
Update to commons-beanutils:1.9.4 without disabling the protection

### PR DESCRIPTION
**Pull request by daniel-beck**
This pull request attempts to update to `commons-beanutils:1.9.4` without disabling the additional protection it offers around the `class` attribute.

It seems in my (fairly limited so far) testing that the problem is `st:include`'s `class` attribute. So this pull request renames the attribute to `clazz` (and keeps the existing setter for compatibility).

This PR additionally proposes a workaround that applies when Jelly files are being parsed and the resulting tags are created: For the `st:include` tag, when the `class` attribute would be set, instead set `clazz`. This attempts to take care of all existing code that sets the `class` attribute for this tag.

I am unsure how best to go about ensuring this doesn't cause a lot of problems. While this fix looks fairly safe, it may also be incomplete. In the short term, #209 seems like the safer approach.

CC @basil

### CODE REVIEW

File: jelly/src/main/java/org/kohsuke/stapler/jelly/AttributeNameRewritingTagScript.java
Comment (basil) [inline jelly/src/main/java/org/kohsuke/stapler/jelly/AttributeNameRewritingTagScript.java:11]:
```suggestion
    private final String original;
```

**Review by basil — APPROVED** [review]
Looks great!

File: jelly/src/main/java/org/kohsuke/stapler/jelly/IncludeTag.java
Comment (basil) [inline jelly/src/main/java/org/kohsuke/stapler/jelly/IncludeTag.java:98]: FYI this warning might be a bit noisy. IIUC plugin maintainers won't be able to migrate to the new syntax until after they update their baseline to the first version of Jenkins core where this is released, since if they need to support older versions of Jenkins core they need to keep using the old syntax in their Jelly files. Besides, I found 1,070 unique instances of `<st:include class=""/>` attributes that needed updating. I doubt that log files and Jira tickets are going to help speed this migration along. Rather I wrote a quick and dirty tool at https://github.com/basil/jelly-migrator/ that can be used once a plugin's Jenkins core baseline is updated to automatically update all Jelly files to the new syntax.

Comment (daniel-beck) [reply jelly/src/main/java/org/kohsuke/stapler/jelly/IncludeTag.java:98]: See https://github.com/jenkinsci/jenkins/pull/5324#issuecomment-790371595

Because of the migration code in `StaplerTagLibrary` I expect there to be absolutely no callers left.

**Review by rsandell — APPROVED** [review]
We've had riskier workarounds in the past.

File: jelly/src/main/java/org/kohsuke/stapler/jelly/IncludeTag.java
Comment (daniel-beck) [reply jelly/src/main/java/org/kohsuke/stapler/jelly/IncludeTag.java]: Added escape hatch that would allow admins to disable this warning.

**Review by MRamonLeon — APPROVED** [review]

**Review by jeffret-b — APPROVED** [review]
This makes good sense and is the best proposal about how to move forward and upgrade to the new version. We can't just allow ourselves to remain on the old version and be blocked from continuing forward.

Hopefully this will work. If it doesn't there is a sufficient amount of escape hatches and configuration to allow people with other scenarios to continue while we correct anything additional that might arise.

**Review by basil — APPROVED** [review]
This looks great! Very nice change.

**Review by oleg-nenashev — APPROVED** [review]
Looks right. Kudos to the Jelly team and @jstrachan for such extensibility

**Review by alecharp — APPROVED** [review]

**Comment by MRamonLeon** [comment]
Who's able to merge this PR @daniel-beck, @oleg-nenashev? It has enough approvals for a few days already. The permission updater repo doesn't mention any maintainer: https://github.com/jenkins-infra/repository-permissions-updater/blob/master/permissions/component-stapler.yml

**Comment by daniel-beck** [comment]
> The permission updater repo doesn't mention any maintainer

That's explained in the linked file in a comment.

@jglick Please hold off releasing for another week until we've merged in changes from the upcoming security update.

**Comment by batmat** [comment]
Why can't we just release and hold-off updating this into Jenkins Core?
We do not understand the rationale here. Thanks!

**Comment by daniel-beck** [comment]
Because there's a privately staged new release of Stapler. That release you're asking for would never be in a core update anyway (and if it were it'd be a pretty big regression over next week's core release for absolutely no reason).

**Pull request merged**
